src/components/MenuBars/MenuBars.tsx

```diff
@@ -129,10 +131,21 @@ export const MenuBars = ({showPreviousColumn, showNextColumn, onPreviousColumn,
           {isAdmin && (
             <ul className="menu__items">
               <li>
-                <TooltipButton direction="left" label="Timer" onClick={showTimerMenu} icon={TimerIcon} />
+                <TooltipButton
+                  direction="left"
+                  label="Timer"
+                  onClick={showTimerMenu}
+                  icon={TimerIcon}
+                  hotkeyHint={
```

Comment:
this one even overlaps the buttons tooltip

Reply:
It should look like this. But I'll look into that
<img width="198" alt="Bildschirmfoto 2022-11-04 um 17 29 18" src="https://user-images.githubusercontent.com/58441634/200027174-2a54e566-4907-457f-85bb-7620cbf39d09.png">